doc/rawdata.rst

```diff
@@ -0,0 +1,31 @@
+.. highlight:: rest
+
+********
+Raw Data
+********
+
+Raw Files
+=========
+
+The raw files may be located anywhere on the harddrive.
+We recommend you organize them by date of observation.
+
+You will need to provide the full path to the files
+and a filename root (unless you wish to specify them
+one by one!).  Here is an example path+root::
+
+    /Users/xavier/Keck/LRIS/data/2016apr06/Raw/LB.20160406.
```

Comment:
Not if one is using pypit_setup, so I fear this is a bit confusing.
Will try to improve.